src/classes/RD_RecurringDonations_TDTM.cls

```diff
@@ -86,6 +86,11 @@ public class RD_RecurringDonations_TDTM extends TDTM_Runnable {
                      r.npe03__Open_Ended_Status__c != system.label.npe03.RecurringDonationClosedStatus)) {
                     r.addError(system.label.npe03.RecurringDonationTooManyInstallmentsError);
                 }            
+                // Reset this field to false if the installment period is not Monthly or Quarterly
```

Comment:
@davidhabib I hadn't thought of this, but I'm with you... I think the 'last day of the month' functionality should be supported for any interval that is based on months.